### PR TITLE
feat: add support for malaysia itn

### PR DESCRIPTION
**Pull request by gustav**

### CODE REVIEW

File: stdnum/my/itn.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    """Check if the number is a valid Malaysian ITN number. This checks the length
+    and formatting."""
+    number = compact(number)
+
+    if _individual_itn_re.match(number) or _non_individual_itn_re.match(number):
+        return number
+    else:
+        if not (10 <= len(number) <= 13):
+            raise InvalidLength()
+        raise InvalidFormat()
```
Comment (EmberCraze) [inline stdnum/my/itn.py:76]: There is no need for else in this case, the if does an early exit

**Review by EmberCraze — CHANGES_REQUESTED** [review]
Some small changes to the checks

File: stdnum/my/itn.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+def validate(number):
+    """Check if the number is a valid Malaysian ITN number. This checks the length
+    and formatting."""
+    number = compact(number)
+
+    if _individual_itn_re.match(number) or _non_individual_itn_re.match(number):
+        return number
+    else:
+        if not (10 <= len(number) <= 13):
```
[the remark this answers is not in the document]
Comment (gustav) [reply stdnum/my/itn.py]: I realise we used an outdated document for the formats. Have updated with the new version.

File: stdnum/my/__init__.py [hunk 3]
```diff
@@ -19,3 +19,8 @@
 # 02110-1301 USA
 
 """Collection of Malaysian numbers."""
+
+# provide aliases
+from stdnum.sg import itn as vat  # noqa: F401
+from stdnum.sg import itn as business_tin  # noqa: F401
```
[the remark this answers is not in the document]
Comment (gustav) [reply stdnum/my/__init__.py:25]: the itn is the business tin, will use nric for individuals

**Pull request merged**